kubernetes/samples/scripts/domain-lifecycle/startServer.sh

```diff
@@ -0,0 +1,144 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a WebLogic managed server in a domain by patching
+  it's 'serverStartPolicy' field to 'ALWAYS'. It also increases the replica
+  count value for the managed server's cluster by '1'.  The replica count
+  value can be kept constant by using '-k' option. If the server pod is
+  already running, script exits without updating 'serverStartPolicy' or
```

Comment:
Fully qualify the start policy & replica count.

Reply:
Fixed.